test/config/utility.cc

```diff
@@ -258,21 +288,24 @@ void ConfigHelper::addSslConfig() {
       TestEnvironment::runfilesPath("/test/config/integration/certs/serverkey.pem"));
 }
 
-void ConfigHelper::loadHttpConnectionManager(
-    envoy::api::v2::filter::network::HttpConnectionManager& hcm) {
+envoy::api::v2::Filter* ConfigHelper::getFilterFromListener() {
```

Comment:
Will we want to parameterize this with the listener number at some point soon?

Reply:
I had the same thought when I wrote it, but realized there was no clear need given it's private.  If we ever do we can default 0, 0, 0.